packages/eas-cli/src/commands/submit.ts

```diff
@@ -89,6 +91,11 @@ export default class Submit extends EasCommand {
       default: false,
       description: 'Enable verbose logging for the submission process',
     }),
+    groups: Flags.string({
+      description: 'Testing groups to send the build to (iOS only).',
```

Comment:
```suggestion
      description: 'Testing groups to send the build to (iOS only)',
```
I don't think other descriptions end with a full stop?